### ISSUE TITLE
Falling tests on throwing error

### ISSUE THREAD

**Issue by YetiOr** [bem/bem-walk#23]
Turned off two falling tests. Because tests stops after first assertion. I think that's not very user-friendly.
```
naming
    ✓ must support original naming 
    ✓ must support Convention by Harry Roberts 
    ✓ must support custom naming 
    ✓ must support several naming 

  flat scheme
    ✓ must end if levels is empty 
    - must throw error if levels is not found
    ignore
      ✓ must ignore entity dir 
      ✓ must ignore entity without ext 
      ✓ must support invalid BEM-notation 
    detect
      ✓ must detect block 
      ✓ must support complex tech 
      ✓ must detect bool mod 
      ✓ must detect mod 
      ✓ must detect elem 
      ✓ must detect bool mod of elem 
      ✓ must detect elem mod 
      ✓ must support few levels 

  multi scheme
    ✓ must support several schemes 

  nested scheme
    ✓ must end if levels is empty 
    - must throw error if levels is not found
    ignore
      ✓ must ignore entity dir 
      ✓ must ignore entity without ext 
      ✓ must support invalid BEM-notation 
      ✓ must ignore file in root of level 
      ✓ must not detect block if filename not match with dirname 
      ✓ must not detect elem if filename not match with dirname 
      ✓ must not detect mod if filename not match with dirname 
      ✓ must not detect elem mod if filename not match with dirname 
    detect
      ✓ must detect block 
      ✓ must support complex tech 
      ✓ must detect bool mod 
      ✓ must detect mod 
      ✓ must detect elem 
      ✓ must detect bool mod of elem 
      ✓ must detect elem mod 
      ✓ must detect complex entities 
      ✓ must support few levels 


  35 passing (97ms)
  2 pending
```

**Comment by YetiOr** [comment]
commit dcfeaf5e5a8eec541ad00e45d8dd1183f991dd92
node -v
v0.11.16

**Comment by andrewblond** [comment]
`mock-fs@2.3.2` works correctly only for `node@0.11.13`.

This problem was fixed in `mock-fs@2.5.0` — https://github.com/tschaub/mock-fs/issues/17.
